lib/src/main/java/vmath/Mat4.java

```diff
@@ -44,8 +44,22 @@ public Mat4(float[] m) {
 
     public float get(int i, int j) {
         if (i < 0 || i >= 4 || j < 0 || j >= 4) {
-            throw new IndexOutOfBoundsException();
+            throw new IndexOutOfBoundsException("Invalid indices");
         }
         return m[i * 4 + j];
-    }
-}
+    } 
+
+    public Mat4 mul(Mat4 other) {
+        Mat4 result = new Mat4();
```

Comment:
This implementation is good. But we follow the [code style rules](https://github.com/toksaitov/vmath/wiki/Code-Style). You may rewrite the implementation like [this](https://github.com/toksaitov/vmath/pull/60/files) or simply follow the rules